launch_testing/launch_testing/legacy/__init__.py

```diff
@@ -0,0 +1,268 @@
+# Copyright 2018 Open Source Robotics Foundation, Inc.
```

Comment:
Was this moved from somewhere? If not, then 2019.

Reply:
So, it was moved from `launch_testing/launch_testing/__init__.py`, which I significantly refactored in 2019 but was originally written in 2018.